static/css/base.css

```
@@ -1835,283 +1835,283 @@ small.hint {
+
+.pricing-table {
+    width: 100%;
+    border-collapse: collapse;
+    min-width: 360px;
+}
+
+.pricing-table th,
+.pricing-table td {
+    padding: 0.95rem 1rem;
+    text-align: left;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)